src/firebase/FirebaseAuthProvider.js

```diff
@@ -25,69 +23,34 @@ const AuthProvider = ({ children }) => {
         return () => unsubscribeFromAuth()
     })
 
-    const authSignIn = (email, password) => new Promise((resolve, reject) => {
-        if (!authExists) {
-            reject('Auth not intitialised.');
-            return;
-        }
+    // all the following functions return promises, not objects or functions. 
+    // there may be an issue with calls being made before auth is initialsed.
```

Comment:
Need to ensure auth is initialized - either use state variable, or test in UI code.